sched/signal/sig_timedwait.c

```diff
@@ -406,7 +406,7 @@ int nxsig_timedwait(FAR const sigset_t *set, FAR struct siginfo *info,
            * that we were waiting for?
            */
 
-          if (sigismember(set, rtcb->sigunbinfo.si_signo))
+          if (nxsig_ismember(set, rtcb->sigunbinfo.si_signo))
```

Comment:
```suggestion
          ret = nxsig_ismember(set, rtcb->sigunbinfo.si_signo);
          if (ret == 1)
```
Plus handling for the case where ret < 1

Reply:
Nevermind here.  Lookat the code, you can see that there is a good signal number check:

    403       if (GOOD_SIGNO(rtcb->sigunbinfo.si_signo))
    404         {
    409           if (sigismember(set, rtcb->sigunbinfo.si_signo))

This might be the case with others as well.